src/main/java/org/apache/maven/plugins/changes/issues/IssuesReportRenderer.java

```diff
@@ -153,89 +139,81 @@ private void constructHeaderRow(Sink sink, List<Issue> issueList, ResourceBundle
         sink.tableRow_();
     }
 
-    private void constructDetailRows(Sink sink, List<Issue> issueList, ResourceBundle bundle, Locale locale) {
-        if (issueList == null) {
-            return;
-        }
+    private void constructDetailRows() {
+
+        // Always use the international date format as recommended by the W3C:
+        // http://www.w3.org/QA/Tips/iso-date
+        // This date format is used in the Swedish locale.
```

Comment:
Why does this (Swedish) matter? It feels like a minor coincidence not relevant to the code.

Reply:
copy paste 😄